embedded/sql/row_reader.go

```diff
@@ -297,24 +298,24 @@ func (r *rawRowReader) ScanSpecs() *ScanSpecs {
 	return r.scanSpecs
 }
 
-func (r *rawRowReader) Columns() ([]ColDescriptor, error) {
+func (r *rawRowReader) Columns(ctx context.Context) ([]ColDescriptor, error) {
```

Comment:
is context required here if it is not being used or propagated?

Reply:
this method is declared in an interface, some concrete implementations are using it. Ideally I'd remove the context on this particular method but it may involve some further changes we may do in the future